pdp_util/raster.py

```diff
@@ -68,8 +69,11 @@ def __call__(self, environ, start_response):
                     content_type='application/json',
                     charset='utf-8')
             return res(environ, start_response)
+
         else:
-            return super(RasterServer, self).__call__(environ, start_response)
+            orca_url = build_orca_url(self.config['handlers'], self.config['thredds_root'], req)
+            return Response(status_code=301, location=orca_url)
```

Comment:
`301` seems to make a fair amount of sense since we are calling a different service from within the `pdp`.